.github/workflows/ci.yml

```diff
@@ -70,6 +70,30 @@ jobs:
         # See tools/ci/src/main.rs for the commands this runs
         run: cargo run -p ci
 
+  check-benches:
+    runs-on: ubuntu-latest
+    needs: ci
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/bin/
+            ~/.cargo/registry/index/
+            ~/.cargo/registry/cache/
+            ~/.cargo/git/db/
+            target/
+          key: ${{ runner.os }}-cargo-check-benches-${{ hashFiles('**/Cargo.toml') }}
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          components: rustfmt, clippy
```

Comment:
Do rustfmt and clippy need to be pulled in?
They are not used afterall.

Reply:
no, no they don't